src/ui.js

```diff
@@ -0,0 +1,244 @@
+// UI Class: Handles UI Tasks
+import Store from './store';
+import Task from './task';
+import taskCompleted from './checkboxes';
+import removeTask from './removeTask';
+import updateTask from './updateTask';
+import clearCompleted from './clearCompleted';
+
+export default class UI {
+  static addApp() {
+    this.addTitle();
+    this.addForm();
+    this.addEmptyUL();
+    const todos = Store.getTasks();
+    this.addTasksUI(todos);
+    this.addbottombtn();
+  }
+
+  static addTitle() {
+    const appDiv = document.querySelector('#appDiv');
+
+    const div4title = document.createElement('DIV');
+    div4title.className = 'd-flex justify-content-start align-items-center border-bottom border-2 px-2 appItem';
+
+    const title = document.createElement('p');
+    title.className = 'fs-5 m-0';
+    title.innerText = "Today's To Do";
+
+    const icon = document.createElement('ICON');
+    icon.className = 'fas fa-sync-alt ms-auto p-2';
+
+    div4title.appendChild(title);
+    div4title.appendChild(icon);
+
+    appDiv.appendChild(div4title);
```

Comment:
This is an example of the lines you could try to replace if you would like to try something different. 
I'll leave a link so you can take a look. [documentation](https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentHTML)